### ISSUE TITLE
False positive from `prefer-switch`

### ISSUE THREAD

**Issue by crystalfp** [sindresorhus/eslint-plugin-unicorn#2759]
In the code below the prefer-switch rule suggests to change the if chain into a switch. But the ifs arguments refer to different variables so cannot be converted to a switch.

prefer-switch

```js
if(n === 0) k = -1;
else if(m === 0) j = -1;
else if(ll === 0) i = -1;
```
